src/core/libmaven/mavenparameters.cpp

```diff
@@ -168,12 +212,16 @@ void  MavenParameters::setPeakDetectionSettings(const char* key, const char* val
     if(strcmp(key,"chargeMax") == 0)
         maxCharge = atof(value);
 
-    if(strcmp(key, "dbDetection") == 0 )
+    if(strcmp(key, "dbDetection") == 0 );
         //TODO
 
-        //TODO: replace with compoundMassCutoffWindow. replace compoundPPMWindow in default_settings_xml. regenerate settings.h
-//     if(strcmp(key, "compoundPPMWindow") == 0 )
-//         compoundPPMWindow = atof(value);
+     if(strcmp(key, "compoundMassCutoffWindow") == 0 ) {
```

Comment:
compoundMassCutoffWindow is being used above also?

Reply:
where ? other than here, it's set in constructor.

Reply:
I confused between type and window